doc/build/dts/api-usage.rst

```diff
@@ -50,6 +50,8 @@ By alias
    :zephyr:code-sample:`blinky`, which uses the ``led0`` alias) that need to
    refer to *some* device of a particular type ("the board's user LED") but
    don't care which one is used.
+   (You can also use :c:func:`DT_HAS_ALIAS()` to verify whether an alias
```

Comment:
Nit/not directly related to this PR since it's wrongly set elsewhere but it should be `:c:macro:` not `:c:func:`

Reply:
I see, but it seems that this can be consistent with the existing one. I will fix all the rst errors in the document in the near future.